eng/Versions.props

```diff
@@ -183,7 +183,7 @@
     <NUnitVersion>3.11.0</NUnitVersion>
     <NUnit3TestAdapterVersion>3.11.2</NUnit3TestAdapterVersion>
     <NUnitLiteVersion>3.11.0</NUnitLiteVersion>
-    <NunitXmlTestLoggerVersion>2.1.36</NunitXmlTestLoggerVersion>
+    <NunitXmlTestLoggerVersion>2.1.41</NunitXmlTestLoggerVersion>
```

Comment:
This version bump was necessary to be able to make use of the `{framework}` placeholder element in the test result xml name, so that we could report results from each TFM, not just the last one run.